package.json

```diff
@@ -1,15 +1,35 @@
 {
   "name": "leaderboard",
   "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
- Good job done so far, but there are some missing scripts missing in your project. The command `npm start` does not work on your project. Please fix this 🙏.

<img width="443" alt="22ef2ff" src="https://user-images.githubusercontent.com/73607512/225128196-0abc69d4-ad26-4acf-91bf-ff45ba39b054.PNG">

Reply:
Hello, @mckento5 thanks for Highlighting the error, thou on my end it was working correctly. Let me check and fix it.